src/Symfony/Component/Cache/DoctrineProvider.php

```diff
@@ -26,7 +26,7 @@ class DoctrineProvider extends CacheProvider implements PruneableInterface, Rese
 
     public function __construct(CacheItemPoolInterface $pool)
     {
-        trigger_deprecation('symfony/cache', '5.4', '"%s" is deprecated, use "Doctrine\Common\Cache\Psr6\DoctrineProvider" instead.');
+        trigger_deprecation('symfony/cache', '5.4', sprintf('"%s" is deprecated, use "Doctrine\Common\Cache\Psr6\DoctrineProvider" instead.', __CLASS__));
```

Comment:
no need for sprintf, trigger_deprecation does it

Reply:
Ah! Does it not do it in phpunit-bridge then? This is where I saw the plain `%s` - https://github.com/symfony/ux/pull/113/checks?check_run_id=3000104396#step:13:103

Reply:
you still need to add `__CLASS__` as extra argument to trigger_deprecation

Reply:
ah, thanks - got it - sorry about that - should be good now